services/resources/deleteResource.ts

```diff
@@ -0,0 +1,40 @@
+export const deleteResource = async (userId: string, idResource: string) => {
+  console.log(userId, idResource);
+  try {
+    const response = await fetch(
+      `${process.env.NEXT_PUBLIC_BACKEND_URL}/resources/deleteResource/${userId}/${idResource}`,
+      {
+        method: "DELETE",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    );
+
+    const responseData = await response.json();
+    console.log(responseData);
```

Comment:
Again, I consider all the console.log() should be restricted for developing environment. I'm unsure if the final user will use these.

Reply:
delete console